modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4034,6 +4034,27 @@ public IgniteInternalFuture<?> resetCacheState(Collection<String> cacheNames) th
         return initiateCacheChanges(reqs).stream().collect(IgniteCollectors.toCompoundFuture());
     }
 
+    /**
+     * Attempts to register a cache locally if it has not already been registered.
+     *
+     * @param cacheNames Cache names.
+     * @return Future that will be completed when all caches are checked.
+     */
+    public IgniteInternalFuture<?> checkCaches(Collection<String> cacheNames) {
```

Comment:
It seems to me, that the name of the method is meaningless. Check caches for what? Please rename it.

Reply:
Tried to fix.